lib/readers/csv.js

```diff
@@ -189,9 +191,21 @@ class CSVReader extends EventEmitter {
   _normalizeCSVDate (msSinceUnixEpoch) {
     // Dates in the CSV file are represented as milliseconds since the unix epoch,
     // converted to a string.
-    const time = parseInt(msSinceUnixEpoch, 10);
+    const time = parseInt(msSinceUnixEpoch.trim(), 10);
     return new Date(time);
   }
+
+  _normalizeCSVLocale (locale) {
+    return locale && locale.trim();
+  }
+
+  _normalizeCSVEmail (email) {
+    return email && email.toLowerCase();
+  }
```

Comment:
Hmm. Didn't I have to switch from pulling normalizedEmail out of the database, to pulling email? When I ran with fxa.accounts.normalizedEmail, I had lots of spurious changed accounts due to Foo@bar.com vs. foo@bar.com.